deployment/terraform/create.go

```diff
@@ -146,20 +146,21 @@ func (t *Terraform) Create() error {
 		}()
 	}
 
+	// TODO: display the entire cluster info from terraformOutput later
+	// when we have cluster support.
+	mlog.Info("Deployment complete.")
```

Comment:
Although we are going to update this, we should note that we are not storing the errors could possibly come from the loop above. If we have errors, we shouldn't consider as complete, or should we?

Reply:
That's a good point. As @ethervoid mentioned in a previous review, I am going to turn this update loop into separate goroutines and listen for errors from them. This will change a bit more once we start deploying clusters. But yes, eventually, this log line will only be printed when the deployment is fully complete.